frontend/src/utils/projectPage.js

```diff
@@ -52,18 +104,88 @@ export const getKitspaceYAMLJson = async assetsPath => {
   return [res.ok, res.ok ? await res.json() : kitspaceYAMLFields]
 }
 
+export const processedAssets = async assetsPath => {
```

Comment:
Not a good name for a function and doesn't describe what it's doing at all. It's fetching the overall status?

Reply:
Yes, I couldn't come with something better

Reply:
`getAssetsStatus`?

Reply:
It doesn't return the status, it returns a boolean, whether the processor is still progressing or not.

Reply:
`getIsProcessingDone`? kind of horrible but at least descriptive once you get over that.

Reply:
Resolved in b990e54.